gems/pending/spec/spec_helper.rb

```diff
@@ -12,6 +14,8 @@
 # Initialize the global logger that might be expected
 require 'logger'
 $log ||= Logger.new("/dev/null")
+# $log ||= Logger.new(STDOUT)
+# $log.level = Logger::DEBUG
```

Comment:
I'd rather do it in followup PR so this PR doesn't get delayed in "naming".

[I like how this one tells you that it's running and how to turn it off](https://github.com/ManageIQ/manageiq/blob/73abbc6b3eae46d027326c14fb412306b5bb6750/config/initializers/rack_ruby_prof.rb#L1-L4)

[We have another place for the vmdb suite that's very crufty and in need of love](https://github.com/ManageIQ/manageiq/blob/73abbc6b3eae46d027326c14fb412306b5bb6750/spec/support/evm_spec_helper.rb#L1-L10).  Whatever you come up with, I'd hope we can normalize for both gems/pending and the old vmdb spec, hence it's probably best to do in a follow up PR.